object/types.proto

```diff
@@ -228,6 +228,12 @@ message Header {
   // * __NEOFS__TICK_TOPIC \
   //   UTF-8 string topic ID that is used for object notification.
   //   DEPRECATED: attribute ignored by servers.
+  // * __NEOFS__EC_RULE_IDX \
+  //   Index of EC rule in container's `PlacementPolicy.ec_rules` according to
```

Comment:
Like this index. For me in "REP 3 EC 3/1" the EC rule index is 1. But in fact it's 0.